image_proc/src/nodelets/rectify.cpp

```diff
@@ -84,6 +85,7 @@ void RectifyNodelet::onInit()
 
   // Read parameters
   private_nh.param("queue_size", queue_size_, 5);
+  private_nh.param("frame_id_suffix", frame_id_suffix_, std::string(""));
```

Comment:
This name is ambiguous. Could you please instead do something like "rectified_frame_suffix"?

Reply:
Done! I have implemented the change. See [88f848a](https://github.com/ros-perception/image_pipeline/pull/773/commits/88f848a9f33f8776feef01a74c999de78e94c9d8)